spec/chartmogul/retry_spec.rb

```diff
@@ -4,12 +4,13 @@
 
 describe 'chartmogul retry request' do
   let(:url) { 'https://api.chartmogul.com/v1/customers/search?email=no@email.com' }
+  let(:api_base) { 'https://api.chartmogul.com' }
```

Comment:
should be the default value. No?

Reply:
yes, but for this test case, we create a double with specific values.